client/landing/stepper/declarative-flow/internals/steps-repository/site-vertical/index.tsx

```diff
@@ -26,9 +26,9 @@ const SiteVertical: Step = function SiteVertical( { navigation } ) {
 	const translate = useTranslate();
 	const headerText = translate( 'What’s your website about?' );
 	const subHeaderText = translate( 'Choose a category that defines your website the best.' );
-	const isSkipSynonyms = useQuery().get( 'isSkipSynonyms' );
 	const isEnglishLocale = useIsEnglishLocale();
 	const isEnabledFTM = isEnabled( 'signup/ftm-flow-non-en' ) || isEnglishLocale;
+	const isSkipSynonyms = useQuery().get( 'isSkipSynonyms' ) || ! isEnglishLocale;
```

Comment:
Could we use `??` to check the value? Otherwise, we cannot enable synonyms even if we set `isSkipSynonyms=false` in query string for non-English locale

Reply:
Nice catch 🙏